assets/scss/modules/_navigation.scss

```diff
@@ -147,3 +145,146 @@ a.back-to-top-link {
   }
 
 }
+
+/*
+Page Navigation
+
+A page navigation element with a previous and next link.
+
+Markup:
+<nav role="navigation" aria-label="Page navigation">
+  <ul class="page-nav">
+    <li class="page-nav__item page-nav__item--previous">
+      <a class="page-nav__link page-nav__link-previous" title="Navigate to previous part" rel="prev" href="#">
+        <span class="page-nav__label">Previous </span>
+        <span class="page-nav__title">Overview</span>
+      </a>
+    </li>
+    <li class="page-nav__item page-nav__item--next">
+      <a class="page-nav__link page-nav__link-next" title="Navigate to next part" rel="next" href="#">
+        <span class="page-nav__label">Next</span>
+        <span class="page-nav__title">Inheritance Tax reliefs</span>
+      </a>
+    </li>
+  </ul>
+</nav>
+
+Styleguide Navigation.Page Navigation
+*/
+.page-nav {
+  display:block;
+  margin:em(64) 0 0 0;
+  padding:em(12) 0;
+  border-bottom:1px solid $light-grey;
+
+  @include contain-floats;
+
+  @include media(mobile) {
+    margin:em(32) 0 0 0
+  }
+}
+
+.page-nav__item {
+  @include core-19;
+  line-height:1.25;
+  float:left;
+  text-align:right;
+  margin:em(5) 0;
+  padding:0;
+  width:49%;
+
+  @include media(mobile) {
+    @include core-14;
+    line-height:1.14286;
+    float:none;
+    width:100%
+  }
+}
+
+.page-nav__item--previous {
+  float:left;
+  text-align:left;
+}
+
+.page-nav__link {
+  display:block;
+  color:$govuk-blue-colour;
+  text-decoration:none;
+
+  &:hover, &:active {
+    background-color:$shaded-bg-grey;
+  }
+
+  &:focus {
+    outline: em(3) solid $yellow;
+    background-color: transparent;
+
+    @include media(mobile) {
+      outline-width: em(3, 16);
+    }
+  }
+
+  @include media(mobile) {
+    @include core-19;
+  }
+}
+
+.page-nav__link-previous {
+  padding:em(14) 0 em(14) em(48);
+
+  &:before {
+    background:transparent url(../images/icons/arrow-sprite.png) no-repeat em(-20) em(-11);
+    margin:em(-4) 0 0 em(-32);
+    float:left;
+    width:em(30);
+    height:em(38);
+    content:" ";
+
+    @include media(mobile) {
+      font-size: 19px;
+    }
+  }
+}
+
+.page-nav__item--next {
+  float:right;
+  text-align:right;
+}
+
+.page-nav__link-next {
+  padding:em(14) em(48) em(14) 0;
+
+  &:before {
+    background:transparent url(../images/icons/arrow-sprite.png) no-repeat em(-102) em(-11);
+    margin:em(-4) em(-32) 0 0;
+    float:right;
+    width:em(30);
+    height:em(38);
+    content:" ";
+  }
+
+  @include media(mobile) {
+    font-size: 19px;
```

Comment:
> No as it's for mobile widths and there's no core-* with a font size of 19 at a mobile width.

so you can use `core-19` here but it will mean the font size is `16px` due to https://github.com/hmrc/assets-frontend/blob/master/assets/govuk_elements/govuk/public/sass/_typography.scss#L134-L138

IMO would be advisable to stick with `core-19` here